### ISSUE TITLE
Could not log in with a cookie

### ISSUE THREAD

**Issue by edoardop13** [codergator/viner#2]
Hi, I installed all the libraries and ran the script, I logged into chrome and when I run the script I put the "--browser Chrome" option. The problem is that I still get this error "Could not log in with a cookie".
I changed the link to "https://www.amazon.it/gp/vine/"

Also if I can suggest a change it could be to intrude the notifications on telegram

**Comment by nothing-much-coder** [comment]
Hi. I haven't played with the script in a while, but it worked for me on MacOS and Fedora. I think it may be finicky and only work with Firefox.

**Comment by edoardop13** [comment]
With macos and chrome it works great. With ubuntu with both chrome and firefox I don't find any product.
![IMG_20190418_140834](https://user-images.githubusercontent.com/40502451/56360189-2902d500-61e4-11e9-964b-446188c4e821.jpg)

**Comment by nothing-much-coder** [comment]
Yeah, I don't know. I'm getting success running it on MacOS with Firefox:

```
python amazon-vine.py --browser Firefox -w 1
```

**Comment by edoardop13** [comment]
Is there a way to not use browser cookies?

**Comment by shailensobhee** [comment]
Can anyone confirm is this whole script still works in 2020? Seems that on Linux/Windows, the cookies are being encrypted.

If it works on Mac, I'll go get a Mac.
